src/core/store.ts

```diff
@@ -105,9 +105,11 @@ listenerMiddleware.startListening({
 listenerMiddleware.startListening({
   actionCreator: emailCreated,
   effect: (action) => {
-    const [email, campId] = action.payload;
+    const email = action.payload;
     Router.push(
-      `/organize/${email.organization?.id}/projects/${campId}/emails/${email.id}`
+      `/organize/${email.organization?.id}/projects/${
```

Comment:
Why is `organization` optional? That should always be there. Maybe the types are wrong?

Reply:
this must be a mistake! thank you for seeing this!!